.github/workflows/examples-efr32.yaml

```diff
@@ -50,12 +50,6 @@ jobs:
       - name: Checkout submodules
         run: scripts/checkout_submodules.py --shallow --platform efr32
```

Comment:
CI examples cannot be changed just yet - needs a docker file build and use the new version.

Reply:
done